source/api/presentation/3.0/index.md

```diff
@@ -701,9 +730,10 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and
 ```
 
 ##### start
-A link from this Manifest or Range, to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases, by adding a fragment to the end of the Canvas's URI specifing the spatial and/or temporal segment of interest. When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Range. This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually skip past uninteresting content.
 
-The value _MUST_ be a JSON object, which _MUST_ have the `id` and `type` properties.
+A link from this Manifest or Range, to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases. When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Range. This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually navigate to find it.
+
+The value _MUST_ be a JSON object, which _MUST_ have the `id` and `type` properties.  The object _MUST_ be either a Canvas, or a Specific Resource with a Selector and a `source` property where the value is a Canvas.
```

Comment:
This is quite a mouthful now, do we need an example?

Reply:
Yes... but how? A second json-doc, I guess, as each resource can only have one `start`?

Reply:
Yes, I think so, Perhaps change current example to be simple canvas:

```
{ "start": { "id": "https://example.org/iiif/1/canvas3", "type": "Canvas" } }
```

and add a second example with `SpecificResource` following #1387:

```
{
  "start": {
    "type": "SpecificResource",
    "source": {
      "id": "https://example.org/iiif/id1/canvas1",
      "type": "Canvas"
    },
    "selector": {
      "type": "PointSelector",
      "conformsTo": "http://iiif.io/api/annex/annotation/pointSelector/",
      "t": 14.5
    }
  }
}
```

Reply:
Propose to do this in whatever PR fixes #1387, not this one.

Reply:
:+1:

Reply:
👍